tests/test_gui_table_logic.py

```
@@ -87,11 +95,17 @@ def test_sort_toggle_and_stable_sort_behavior():
-    assert [row.record.source_path for row in ordered[:2]] == ["/workspace/2.env", "/workspace/1.env"]
+    if not ([row.record.source_path for row in ordered[:2]] == ["/workspace/2.env", "/workspace/1.env"]):
+        raise AssertionError()
+
 
     toggled = toggle_sort(state, "name")
-    assert toggled.column == "name"
-    assert toggled.descending is True
+    if not (toggled.column == "name"):
+        raise AssertionError()
+
+    if not (toggled.descending is True):
```

Ruff (0.15.2)

[error] Test for object identity should be `is not`

Convert to `is not`

(E714)